apidoc/Titanium/App/iOS/UserDefaults.yml

```diff
@@ -0,0 +1,257 @@
+---
+name: Titanium.App.iOS.UserDefaults
+summary: |
+    The UserDefaults module is used for storing application-related data in property/value pairs 
+    that persist beyond application sessions and device power cycles. UserDefaults allows the suiteName
+    of the UserDefaults to be specified at creation time.
+description: |
+    Unlike Titanium.App.Properties, Titanium.App.iOS.UserDefaults does not pull properties defined in the `tiapp.xml` file.
+
+    To create a notification action, use the <Titanium.App.iOS.createUserDefaults> method.
+creatable: false
+extends: Titanium.Proxy
```

Comment:
I think you could have extended from Titanium.App.Properties, so you didn't have to repeat the docs.